### ISSUE TITLE
Documenting turbo links way of using Senna

### ISSUE THREAD

**Issue by eduardolundgren** [liferay/sennajs.com#33]
Using senna as the body as a surface is very easy to teach and use, https://github.com/liferay/senna.js/tree/master/examples/turbolinks.

We need to add documentation for that. What do you guys think?

**Comment by brunobasto** [comment]
Yeah, totally agree. This is probably the use case people will want more as
its easy to setup.

**Comment by mjbradford89** [comment]
I agree as well.  I've been making some changes to the documentation recently so I can create that section too if that's alright.

It shouldn't require a ton of new information, right? just some examples and explanation of the attributes on the body element and what they do?

**Comment by zenorocha** [comment]
Yep, feel free to update that pull request @mjbradford89.

**Comment by mjbradford89** [comment]
Updated https://github.com/liferay/sennajs.com/pull/32, thanks!

**Comment by zenorocha** [comment]
Merged and deployed 🚀
